translator/toenvconfig/toEnvConfig.go

```diff
@@ -36,6 +37,9 @@ func ToEnvConfig(jsonConfigValue map[string]interface{}) []byte {
 		if isDebug, ok := agentMap[debugKey].(bool); ok && isDebug {
 			envVars[envconfig.CWAGENT_LOG_LEVEL] = "DEBUG"
 		}
+		if awsSDKLogLevel, ok := agentMap[awsSDKLogLevelKey].(string); ok {
+			envVars[envconfig.AWS_SDK_LOG_LEVEL] = awsSDKLogLevel
+		}
```

Comment:
`jsonConigValue` contains the stuff read from the Agent's JSON config.
So this is looking for `"aws_sdk_log_level"` in the Agent's JSON config.
This function returns a `[]byte` which will soon be written out to `env-config.json`.